tests/constraints/tests.py

```diff
@@ -142,6 +142,30 @@ def test_eq_with_condition(self):
             ),
         )
 
+    def test_eq_with_include(self):
+        self.assertEqual(
+            models.UniqueConstraint(
+                fields=['foo', 'bar'], name='include',
+                include=['baz']
```

Comment:
Please add trailing commas or better yet, these can all be condensed to one line I think.